neo/app/controllers/Application.scala

```diff
@@ -59,7 +60,8 @@ class Application @Inject()(val wsClient: WSClient,
   }
 
   def ws = WebSocket.accept[JsValue, JsValue] { request =>
-    ActorFlow.actorRef(out => NeoActor.props(out, wsClient, request.host, config))
+    val prop = BerryClient.props(new JSONParser(), manager, new QueryPlanner(), config)
+    ActorFlow.actorRef(out => RequestRouter.props(out, prop, config))
```

Comment:
one correction, the flow is always `Router` -> `Berry` -> `Out`. 
What I mean is to make the code clearly deliver this message by build the Berry with this `out`.